docs/source/release/v6.4.0/mantidworkbench.rst

```diff
@@ -61,14 +55,21 @@ Sliceviewer
 New Features
 ############
 
-- New cut viewer tool for non-axis aligned cuts in slice viewer (currently only supports 3D MD workspaces where all dimensions are Q).
+- **A new cut viewer tool for non-axis aligned cuts. Please Note: This currently only supports 3D MD workspaces where all dimensions are Q.**
+
+.. image::  ../../images/SliceViewer_CutViewer.png
+            :align: center
+
 
 Bugfixes
 ########
 
-- Color limit autoscaling now works for MDHisto workspaces in non-orthogonal view in sliceviewer.
-- Fixed a bug caused by moving to matplotlib 3.5 that prevented users from zooming in on a plot in SliceViewer.
-
+- Color limit autoscaling now works for ``MDHisto`` workspaces in non-orthogonal view in Sliceviewer.
+- Using the ``Region of Interest`` tool when the colorscale is set to Power will no longer cause a position related error.
+- Using the ``Log`` or ``SymmetricLog10`` colorbar normalisation options on masked data is now disabled to prevent an error.
+- Fixed a bug with peak selection causing a crash for 4D MD workspaces.
+- Peak addition now correctly assigns new peak ``HKL`` in nonorthogonal view.
+- Peak overlay is now disabled when non-Q axis is viewed. The peak object does not have a position defined for non-Q dimension.
```

Comment:
I'm not certain about these suggested changes, but the singulars read weird without them:
```suggestion
- The peak overlay is now disabled when a non-Q axis is viewed. The peak object does not have a position defined for a non-Q dimension.
```